frontend/app/utils/Base64.scala

```diff
@@ -0,0 +1,69 @@
+package utils
+
+/**
+ * Base64 encoder
+ * @author Mark Lister
+ *         (c) Mark Lister 2014
+ *
+ *         I, the copyright holder of this work, release this work into the public domain.
+ *         This applies worldwide. In some countries this may not be legally possible;
+ *         if so: I grant anyone the right to use this work for any purpose, without any
+ *         conditions, unless such conditions are required by law.
+ *
+ *         The repo for this Base64 encoder lives at  https://github.com/marklister/base64
+ *         Please send your issues, suggestions and pull requests there.
+ *
+ */
+
+object Base64 {
```

Comment:
I don't really see the point of adding another dependency just for base 64 encoding. This seems like a nice compact bit of code that doesn't add any extra stuff we won't use